diff --git a/node_modules/epfl-sti-react-library/dist/Navigations/Avatar.js b/node_modules/epfl-sti-react-library/dist/Navigations/Avatar.js
index 5c848d2..250f9f7 100644
--- a/node_modules/epfl-sti-react-library/dist/Navigations/Avatar.js
+++ b/node_modules/epfl-sti-react-library/dist/Navigations/Avatar.js
@@ -9,8 +9,6 @@ var _react = _interopRequireDefault(require("react"));
 
 var _propTypes = _interopRequireDefault(require("prop-types"));
 
-var _sti_logo = _interopRequireDefault(require("./sti_logo.png"));
-
 require("./index.css");
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
@@ -25,7 +23,7 @@ function Avatar(_ref) {
     return "url('".concat(user.photo_url, "')");
   };
 
-  var avatarLogo = logoUrl || _sti_logo.default;
+  var avatarLogo = logoUrl || "";
   var avatarLogoAltText = logoAltText || 'Logo STI, EPFL school of engineering';
 
   function handleLogout() {
