diff --git a/node_modules/@grpc/grpc-js/build/src/constants.js b/node_modules/@grpc/grpc-js/build/src/constants.js
index b1a8e5f..614c2be 100644
--- a/node_modules/@grpc/grpc-js/build/src/constants.js
+++ b/node_modules/@grpc/grpc-js/build/src/constants.js
@@ -60,5 +60,5 @@ var Propagate;
 // -1 means unlimited
 exports.DEFAULT_MAX_SEND_MESSAGE_LENGTH = -1;
 // 4 MB default
-exports.DEFAULT_MAX_RECEIVE_MESSAGE_LENGTH = 4 * 1024 * 1024;
+exports.DEFAULT_MAX_RECEIVE_MESSAGE_LENGTH = 10 * 1024 * 1024;
 //# sourceMappingURL=constants.js.map
\ No newline at end of file
diff --git a/node_modules/@grpc/grpc-js/src/constants.ts b/node_modules/@grpc/grpc-js/src/constants.ts
index 865b24c..96da4f4 100644
--- a/node_modules/@grpc/grpc-js/src/constants.ts
+++ b/node_modules/@grpc/grpc-js/src/constants.ts
@@ -63,4 +63,4 @@ export enum Propagate {
 export const DEFAULT_MAX_SEND_MESSAGE_LENGTH = -1;
 
 // 4 MB default
-export const DEFAULT_MAX_RECEIVE_MESSAGE_LENGTH = 4 * 1024 * 1024;
+export const DEFAULT_MAX_RECEIVE_MESSAGE_LENGTH = 10 * 1024 * 1024;
